sandbox/cri/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM rust:1.32 as base-build
+
+ENV TMP_BUILD_DIR=/tmp/build
+ENV CARGO_HOME=/usr/local/rust
+ENV RUSTUP_HOME=/usr/local/rust
+ENV RUST_VERSION=1.32.0
+ENV PATH="$CARGO_HOME/bin:$PATH"
+
+ARG FIRECRACKER_VERSION="0.15.0"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        curl \
+        make \
+        musl-tools \
+        libseccomp-dev \
+        btrfs-tools \
+        socat \
+        sudo \
+        iptables \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install the Rust toolchain
+#
+# Install 1.32.0 due to Firecracker crashing with 148 code
+RUN mkdir "$TMP_BUILD_DIR" \
+    && curl https://sh.rustup.rs -sSf | sh -s -- --verbose -y --default-toolchain "$RUST_VERSION" \
+    && rustup target add x86_64-unknown-linux-musl \
+    && git clone https://github.com/firecracker-microvm/firecracker.git
+
+WORKDIR firecracker
+RUN git checkout tags/v"$FIRECRACKER_VERSION" \
+    && cargo build --release --features vsock --target-dir "$TMP_BUILD_DIR" \
+    && cp "$TMP_BUILD_DIR"/x86_64-unknown-linux-musl/release/firecracker /usr/local/bin \
+    && cp "$TMP_BUILD_DIR"/x86_64-unknown-linux-musl/release/jailer /usr/local/bin \
+    && mkdir /etc/containerd && touch /etc/containerd/config.toml \
+    && mkdir -p /go/src/github.com/containerd/containerd
+
+from golang:1.12 as cri-build
```

Comment:
While instructions are not case sensitive, the convention is to use uppercase letters.
https://docs.docker.com/v17.09/engine/reference/builder/#format